### ISSUE TITLE
Support data type POINT

### ISSUE THREAD

**Issue by dschinkel** [oguimbal/pg-mem#10]
Can you support this data type?  I need it or else I can't create this table.  I'm just showing the field here which had the problem, removed the rest of the fields because you just need to see that the field type is not supported here

```js
Error: 🔨 Not supported 🔨 : Data type "point"

CREATE TABLE public.community (
    coordinates point
);

```

**Comment by oguimbal** [comment]
Sure, that was on my roadmap. I'll look into it.

**Comment by oguimbal** [comment]
Points are implemented in `pg-mem@1.4.4`  (parser [here](https://github.com/oguimbal/pgsql-ast-parser/commit/e495c6fe105c8545b72ed15ad387852ec5b62f2b), and execution [here](https://github.com/oguimbal/pg-mem/commit/03221199d15f1a74a768722c3ee3261351deb90c))

Other [geometric types](https://www.postgresql.org/docs/9.4/datatype-geometric.html) have also been implemented, but they not yet behave exactly like postgres ones (working on it), so use them with care.

![image](https://user-images.githubusercontent.com/8973947/101906354-1a3b1400-3bb9-11eb-928a-b85f35a3f6ec.png)
